src/main/resources/db/changesets/20221020_duosfirecloudgroupprimarykey.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: duosfirecloudgroupprimarykey
+      author: okotsopo
+      remarks: |
+        On table creation, we made the DUOS dataset ID our sole primary key, and got uniqueness
+        protection for free as a result.
+        But we prefer to mint a surrogate primary key to guard against the possibility that
+        DUOS IDs could be renamed or otherwise represented differently in the future.
+        As we still want to guard against having multiple rows with the same DUOS ID,
+        we must add a uniqueness constraint to the duos_id column.
+      changes:
+        - dropPrimaryKey:
+            tableName: duos_firecloud_group
+            constraintName: duos_firecloud_group_pkey
```

Comment:
Just curious - It doesn't look like you defined a constraint name when creating the primary key. So, is this just the default?

Reply:
Good question.  Yes, this is the default! I checked locally by describing the table prior to applying the changeset.

Before:
```
datarepo=# \d duos_firecloud_group
                             Table "public.duos_firecloud_group"
        Column         |           Type           | Collation | Nullable |      Default      
-----------------------+--------------------------+-----------+----------+-------------------
 id                    | uuid                     |           | not null | gen_random_uuid()
 duos_id               | character varying(36)    |           | not null | 
 firecloud_group_name  | character varying(100)   |           | not null | 
 firecloud_group_email | character varying(256)   |           | not null | 
 created_by            | character varying(256)   |           | not null | 
 created_date          | timestamp with time zone |           | not null | now()
 last_synced_date      | timestamp with time zone |           |          | 
Indexes:
    "duos_firecloud_group_pkey" PRIMARY KEY, btree (duos_id)
```
    
After:

```
    datarepo=# \d duos_firecloud_group                                                                                                                                               Table "public.duos_firecloud_group"
        Column         |           Type           | Collation | Nullable |      Default      
-----------------------+--------------------------+-----------+----------+-------------------
 id                    | uuid                     |           | not null | gen_random_uuid()
 duos_id               | character varying(36)    |           | not null | 
 firecloud_group_name  | character varying(100)   |           | not null | 
 firecloud_group_email | character varying(256)   |           | not null | 
 created_by            | character varying(256)   |           | not null | 
 created_date          | timestamp with time zone |           | not null | now()
 last_synced_date      | timestamp with time zone |           |          | 
Indexes:
    "duos_firecloud_group_pkey" PRIMARY KEY, btree (id)
    "duos_firecloud_group_duos_id_key" UNIQUE CONSTRAINT, btree (duos_id)
```

Reply:
Awesome, thanks!